### PR TITLE
Fix puma version & actic_admin icons

### PR DESCRIPTION
**Pull request by CR1AT0RS**
#### Board:
*  Fixed arctic_admin icons: #260 
*. Updated Puma version to avoid issues in new OSX 10.14 above builds: #259

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -19,7 +19,7 @@ gem 'jbuilder', '~> 2.10'
 gem 'oj', '~> 3.9', '>= 3.9.2'
 gem 'pagy', '~> 3.7', '>= 3.7.5'
 gem 'pg', '~> 1.1', '>= 1.1.4'
-gem 'puma', '~> 4.3'
+gem 'puma', '~> 5.0'
```
Comment (santib) [inline Gemfile:22]: Reminder to run `bundle install` and also commit the `Gemfile.lock` changes.

File: package.json [hunk 2]
```diff
@@ -3,6 +3,7 @@
   "private": true,
   "dependencies": {
     "@activeadmin/activeadmin": "^2.7.0",
+    "@fortawesome/fontawesome-free": "^5.14.0",
```
Comment (santib) [inline package.json:6]: Same here, remember to run `yarn install` and commit the `yarn.lock`

**Review by santib — COMMENTED** [review]
Hey @CR1AT0RS , thanks for opening the issues and PR ❤️ 

I left some comments, once that's fixed I'm good to merge 😄

**Comment by CR1AT0RS** [comment]
Done! @santib

**Review by santib — APPROVED** [review]

**Review by jpascual1994 — APPROVED** [review]
Thanks for the PR 😁

**Pull request merged**